### PR TITLE
Migrate add to bookmark button to the new design - Closes #2114

### LINKED ISSUE #2114: Migrate add to bookmark button to the new design
The goal is to implement add to the bookmark button on the wallet page. The designs are ready in [InVision](https://projects.invisionapp.com/share/6STGFKRXRQM#/screens/384055163).

### Design requirements
- Input field should remain `read-only` unless the field is clicked upon.
- Confirmation button ("Save changes") should appear only when the input value is changed.
- Confirmation button should fade in using a similar animation as feedback text.
- Dialog should close after any action (save, edit, remove).
- Space above and below feedback text should be equal.
- Dropdown external padding should be aligned with design
- Delegates can be bookmarked but without the possibility of editing the bookmark name - their delegate name will stay their bookmark name.

### PR DESCRIPTION
**Pull request by colado**
### What issue have I solved?

#2114 / [inVision screens](https://projects.invisionapp.com/share/6STGFKRXRQM#/screens/384055163)

### How have I implemented/fixed it?
- Updated `bookmarkDropdown` with edit bookmark functionality.
- Added and applied `WarningButton`.

### How has this been tested?
Bookmarks should be edited and its result should be apparent in the bookmark list.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js [hunk 1]
```diff
@@ -200,38 +229,36 @@ class Bookmark extends React.Component {
...
-        {isBookmark
-          ? (
-            <PrimaryButton
+        {isBookmark ? (
+          <React.Fragment>
+            <div className={`${styles.editButtonContainer} ${hasValueChanged ? styles.show : styles.hide}`}>
+              <PrimaryButton
+                className="bookmark-button"
+                disabled={!fields.accountName.value}
+                size="xs"
```
Comment (slaweet) [inline src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js]:
```suggestion
                size="s"
```
Based on the design

File: src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js [hunk 2]
```diff
@@ -200,38 +229,36 @@ class Bookmark extends React.Component {
...
+                onClick={this.handleBookmark}
+              >
+                {t('Save changes')}
+              </PrimaryButton>
+            </div>
+            <WarningButton
               className="bookmark-button"
               size="xs"
```
Comment (slaweet) [inline src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js]:
```suggestion
                size="s"
```
Based on the design

File: src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js [hunk 3]
```diff
@@ -191,6 +219,7 @@ class Bookmark extends React.Component {
               name="accountName"
               value={fields.accountName.value}
               placeholder={t('ie. Lisker123')}
+              onClick={this.handleInputClick}
               readOnly={fields.accountName.isReadOnly}
               className={`${styles.input} ${fields.accountName.error ? 'error' : ''}`}
               feedback={accountName.feedback}
```
Comment (slaweet) [inline src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js]: The input should have
```
                size="xs"
```
Based on the design.

File: src/components/transactions/transactionsOverviewHeader/transactionsOverviewHeader.js [hunk 4]
```diff
@@ -68,8 +68,8 @@ class transactionsHeader extends React.Component {
             <DropdownButton
               buttonClassName="bookmark-account-button"
               className={`${styles.bookmarkDropdown} bookmark-account`}
-              buttonLabel={isBookmark ? t('Account bookmarked') : t('Bookmark account')}
-              ButtonComponent={isBookmark ? SecondaryButton : PrimaryButton}
+              buttonLabel={isBookmark ? t('Edit bookmark') : t('Bookmark')}
+              ButtonComponent={PrimaryButton}
```
[the remark this answers is not in the document]
Comment (colado) [reply src/components/transactions/transactionsOverviewHeader/transactionsOverviewHeader.js]: Thank you for the feedback, @slaweet. I've fixed all the issues mentioned above, plus there were [a couple new requirements from design](https://github.com/LiskHQ/lisk-hub/issues/2114) that are implemented now. Let me know if you encounter any other problems.

File: src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js (hunk 1 above)
Comment (colado) [reply src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js]: ✅

File: src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js (hunk 2 above)
Comment (colado) [reply src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js]: ✅

File: src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js (hunk 3 above)
Comment (colado) [reply src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js]: ✅

**Comment by slaweet** [comment]
Everything works well now.

Once the test coverage on Jenkins will pass, I can approve.
```
Jest: "/home/lisk/workspace/lisk-hub_PR-2474/src/components/transactions/transactionsOverviewHeader/transactionsOverviewHeader.js" coverage threshold for functions (80%) not met: 75%
```
https://jenkins.lisk.io/blue/organizations/jenkins/lisk-hub/detail/PR-2474/8/pipeline

**Comment by colado** [comment]
Thank you, @imdominic. I implemented the changes and Jenkins is passing. Could you please review the last changes, @slaweet?

**Comment by colado** [comment]
Thank you, @slaweet, I have fixed the issue. Can you please check again?

**Pull request merged**